server/server/views/registration.py

```diff
@@ -54,5 +55,5 @@ def confirm_registration_view(request):
         url = 'localhost:6543/users/{}'.format(user)
```

Comment:
Static URL isn't good. You can use current_route_url from pyramid.url.
[https://docs.pylonsproject.org/projects/pyramid/en/latest/api/url.html#pyramid.url.current_route_url](url)